src/components/Nav/Navigation.tsx

```diff
@@ -36,7 +37,7 @@ class Navigation extends React.Component {
 
   render() {
     return (
-      <div>
+      <DivlessWrapper>
```

Comment:
Won’t <> and </> do the same?

Reply:
https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html

Reply:
Hmm. I didn't know about <> and </>

Reply:
fwiw, <DivlessWrapper> looks less like a mistake than </>

Reply:
Seems that `<> stuff</> ` is just a shorter version of  `<React.Fragment>stuff<React.Fragment/>`

https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html#jsx-fragment-syntax